src/dotfile.c

```diff
@@ -182,11 +182,13 @@ set_conf(const char *key, const char *val)
 	int ret = 0;
 	char *tmpc = NULL;
 
-	if (strcmp(key, "light_colors") == 0) {
+	if (strcmp(key, "bold_font") == 0) {
 		if ((tmp = is_boolean_value(val)) == -1)
 			goto unknown_boolean_value;
-		else
-			cnf.lightcol = tmp;
+		else if (tmp == 1)
+			cnf.lightcol = 1;
```

Comment:
This is not exactly what I was thinking about. Why this 1? I'd like to avoid magic numbers if possible.